main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json", "r") as file:
+            players_data = json.load(file)
```

Comment:
Consider adding error handling for JSON parsing errors, such as using a try-except block to catch `json.JSONDecodeError`.